package.json

```diff
@@ -8,10 +8,24 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "start": "nodemon server/server.js",
+    "build": "webpack --config webpack.config.js --watch",
+    "seed": "node seeding.js"
   },
   "dependencies": {
+    "express": "^4.16.4",
+    "mysql2": "^1.6.5",
```

Comment:
For which feature did you use mysql2 instead of sticking with the "mysql" package?

Reply:
I tried using mysql, but sequelize was giving me an error and said to use mysql2 instead.